### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by PromiseXu1** [DuanYiqun/pytorch_implementation_of_Yolov1#1]
Hi, thanks for your code firstly!  I‘m running it , but i don't know what the results it will be shown like. Is there any test image with bboxs?  Or just prints the loss and train time ?
